sct_ssh.py

```diff
@@ -127,12 +129,12 @@ def gce_get_proxy_command(instance: compute_v1.Instance, strict_host_checking: b
         target_ip = list(gce_public_addresses(instance))[0]
         proxy_command = ''
         target_username = 'scylla-test'
-        target_key = '~/.ssh/scylla-test'
     return proxy_command, target_ip, target_username, target_key
 
 
 def aws_get_proxy_command(instance: dict, force_use_public_ip: bool, strict_host_checking: bool = False) -> [str, str, str]:
     aws_region = AwsRegion(get_region(instance))
+    target_key = f'~/.ssh/{SSH_KEY_AWS_DEFAULT}'
```

Comment:
this will be bit disturbing when connecting to test runs exectuded from different branches if we don't backport.
I'd propose to backport it to 6.0 when it works ok for some time. We could also prepare PR for older branches to add new keys to `~/.ssh/authorized_keys` only, but I'm not sure if it is worth the effort.

Reply:
one would need to use the relevant branch

other option which is a bit longer, is to put is as tag on the instances.

I think it's a bit of an overkill, we can extend the commandlines to have option to select the key on you own, it makes a bit more sense.